nemo/core/classes/optimizers.py

```diff
@@ -0,0 +1,281 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
let's put this under nemo.core.optim.Novograd.py and nemo.core.optim.optimizers.py

__all__  accessible from nemo.core.optim.*